public/sw.js

```diff
@@ -12,22 +12,27 @@ self.addEventListener('install', event => {
 
 const fetchEventHandler = async event => {
   if (
-    /^https:\/\/ichef(\.test)?\.bbci\.co\.uk\/(news|ace\/(standard|ws))\/.+(\.jpg|\.png)$/.test(
+    /^https:\/\/ichef(\.test)?\.bbci\.co\.uk\/(news|images|ace\/(standard|ws))\/.+.webp$/.test(
       event.request.url,
     )
   ) {
     const req = event.request.clone();
 
     // Inspect the accept header for WebP support
-    let supportsWebp = false;
-    if (req.headers.has('accept')) {
-      supportsWebp = req.headers.get('accept').includes('webp');
-    }
 
-    // If we support WebP
-    if (supportsWebp && !/\/amz\/worldservice\/.*/.test(event.request.url)) {
+    const supportsWebp =
+      req.headers.has('accept') && req.headers.get('accept').includes('webp');
+
+    // if supports webp is false in request header then don't use it?
+    // If we support WebP don't remove
+    // if it doesn't support, remove
+    // look at the headers Accept image/webp,*/
+    // downgrade/remove if not there
```

Comment:
```suggestion
    // if accept header doesn't indicate support for webp remove .webp extension
```
Found the comment a bit difficult to follow, tbh I feel the code probably documents itself here as per our coding standards guidance: https://github.com/bbc/simorgh/blob/289302ede274dada710f911004ba168d0ce24802/docs/Coding-Standards/Clean-Code.mdx#code-commenting

Reply:
It was mostly notes for myself I made near the beginning to clarify to myself! Fine to remove them